query/config.go

```diff
@@ -218,8 +218,22 @@ func (t ConfigSummaryRequest) healthClause() []clause.Expression {
 }
 
 func (t *ConfigSummaryRequest) filterClause(q *gorm.DB) *gorm.DB {
+	var orClause *gorm.DB
 	for k, v := range t.Filter {
-		q = q.Where("config_items.labels @> ?", types.JSONStringMap{k: v})
+		if strings.HasPrefix(v, "!") {
```

Comment:
Can we not use the same code as the middleware - i.e. to support multiple values